src/main/java/com/laserfiche/api/client/model/OffsetDateTimeDeserializer.java

```diff
@@ -0,0 +1,32 @@
+package com.laserfiche.api.client.model;
+
+import com.fasterxml.jackson.core.JacksonException;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.TreeNode;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer;
+import org.threeten.bp.OffsetDateTime;
+
+import java.io.IOException;
+
+public class OffsetDateTimeDeserializer extends StdDeserializer<OffsetDateTime> {
+    public OffsetDateTimeDeserializer() {
+        this(null);
+    }
+
+    public OffsetDateTimeDeserializer(Class<?> vc) {
```

Comment:
would you please rename vc to something more meaningful?
I couldn't guess what it stands for.

Reply:
Yeah, I think this is copied from the repository client project. We probably also need a better name there.

Reply:
done